### PR TITLE
Monitoring functions for bash script - Fixes

### PR DESCRIPTION
**Pull request by bozturkMSFT**
Contains the fixes that didn't make it into https://github.com/dotnet/install-scripts/pull/70

### CODE REVIEW

File: src/MonitoringFunctions.Common/InstallScriptRunner.cs [hunk 1]
```diff
@@ -54,8 +53,12 @@ public static ScriptDryRunResult ParseDryRunOutput(string? output)
 
             ScriptDryRunResult result = new ScriptDryRunResult();
 
-            using StringStream stringStream = new StringStream(output);
-            using StreamReader streamReader = new StreamReader(stringStream);
+            if(output == null)
```
Comment (ladipro) [inline src/MonitoringFunctions.Common/InstallScriptRunner.cs:56]: super-nit: Missing space before `(`.

File: src/MonitoringFunctions.Linux/DryRunUrlChecker.cs [hunk 2]
```diff
@@ -7,7 +7,7 @@
 namespace MonitoringFunctions.Linux
 {
     /// <summary>
-    /// Runs the shell script in -DryRun mode and checks weather the generated links are accessible
+    /// Runs the shell script in -DryRun mode and checks whether the generated links are accessible
```
Comment (ladipro) [inline src/MonitoringFunctions.Linux/DryRunUrlChecker.cs:10]: nit: I believe that the typo exists in the Windows version of this file as well.

**Review by ladipro — APPROVED** [review]
Thank you!

**Comment by bozturkMSFT** [comment]
Auto-publishing the functions to the linux app requires a service connection which hasn't been created yet.
I will merge this once it is ready.

**Comment by bozturkMSFT** [comment]
Service connection has been created.

**Review by AR-May — APPROVED** [review]
LGTM

**Pull request merged**